pkg/probe/http/http.go

```diff
@@ -120,21 +120,21 @@ func DoHTTPProbe(url *url.URL, headers http.Header, client GetHTTPInterface) (pr
 	b, err := utilio.ReadAtMost(res.Body, maxRespBodyLength)
 	if err != nil {
 		if err == utilio.ErrLimitReached {
-			klog.V(4).Infof("Non fatal body truncation for %s, Response: %v", url.String(), *res)
+			klog.V(4).InfoS("Non fatal body truncation for URL", "url", url.String(), "res", *res)
 		} else {
 			return probe.Failure, "", err
 		}
 	}
 	body := string(b)
 	if res.StatusCode >= http.StatusOK && res.StatusCode < http.StatusBadRequest {
 		if res.StatusCode >= http.StatusMultipleChoices { // Redirect
-			klog.V(4).Infof("Probe terminated redirects for %s, Response: %v", url.String(), *res)
+			klog.V(4).InfoS("Probe terminated redirects for URL", url.String(), *res)
```

Comment:
```suggestion
			klog.V(4).InfoS("Probe terminated redirects for URL", "url", url.String(), "response", *res)
```

for consistency with other logs

Reply:
Fixed.